src/api/test-controller/index.js

```diff
@@ -41,6 +41,8 @@ import {
 import { WaitCommand, DebugCommand } from '../../test-run/commands/observation';
 import assertRequestHookType from '../request-hooks/assert-type';
 
+const promiseThen = Promise.resolve().then;
```

Comment:
I don't like. In general case, you should deal with multiple Promise implementations (Node.js native, pinkie, bluebird, etc,). You shouldn't override a Promise method with a method from some predefined Promise type.

Reply:
But it is the way how it works now, I've just fix it

Reply:
We've discussed it on private, and unfortunately I can't find a better way right now. I'll try to refactor it in context of #1859.